.env.example

```diff
@@ -0,0 +1,32 @@
+#
+# EXAMPLE ENVIRONMENT
+#
+# When setting up the app, copy this .env.example file to .env and fill in the
+# values below
+#
+# WARNING: ANY VALUES PROVIDED HERE WILL BE SUBSTITUTED VERBATIM WHEN BUILDING
+# THE CODE DISTRIBUTED AS PART OF THE FRONT-END BUNDLE
+#
+# DO NOT ENTER ANY PRIVATE/ADMIN API KEYS OR OTHER SECRET DETAILS
+#
+
+
+# 
+# A search-only or secured API Key created on Algolia
+#
+# REPEAT WARNING: DO NOT USE AN ADMIN KEY, AS THIS VALUE WILL BE INCLUDED IN
+# THE FRONT-END CODE
```

Comment:
I think it would be really helpful to include the link that you included in the PR as part of the comment (https://www.algolia.com/doc/guides/building-search-ui/going-further/api-keys-security/react/)